packages/accordion/src/Default/Default.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Accordion, ThemeProvider } from 'react-bootstrap';
+import { AccordionDefaultProps, themePrefixesProps } from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+
+export const Component = (props: AccordionDefaultProps) => {
+  const themePrefixes: themePrefixesProps = {};
+  const baseClass = 'accordion';
+  const { items } = props;
+  const prefix = props.prefix || '';
+  const [isActive, setIsActive] = useState<string[]>([]);
+
+  const locals = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'theme',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    ['prefix', 'theme', 'size', 'items']
+  );
+
+  themePrefixes[baseClass] = `owlui-${baseClass}`;
+  themePrefixes['accordion-button'] = 'owlui-accordion-button';
+  themePrefixes['accordion-body'] = 'owlui-accordion-body';
+  themePrefixes['accordion-collapse'] = 'owlui-accordion-collapse';
+  themePrefixes['accordion-item'] = 'owlui-accordion-item';
+  themePrefixes['accordion-collapsing'] = 'owlui-accordion-collapsing';
+  themePrefixes['collapsing'] = 'owlui-collapsing';
+  // themePrefixes['collapsed'] = 'owlui-collapsed';
+  themePrefixes['show'] = 'owlui-show';
+
+  const toggleActive = (event: React.MouseEvent<HTMLElement>) => {
+    if (!isActive.includes(event.currentTarget.dataset.index as string)) {
+      setIsActive([...isActive, event.currentTarget.dataset.index as string]);
+    } else {
+      setIsActive(
+        isActive.filter(e => e !== event.currentTarget.dataset.index)
+      );
+    }
+  };
+
+  return (
+    <ThemeProvider prefixes={themePrefixes}>
+      <Accordion {...locals} style={{ width: '70vw' }}>
```

Comment:
I see what you mean, but moving it to the wrapper has the effect of narrowing the container around the accordion, rather than the accordion itself. It was really only there for dev purposes anyway (so that the accordion remains a given size when you open and close the child elements). For now I've replaced the viewport-based width with 20em but we can just remove it if that's better